### DISCUSSION TITLE
The display of ASCII art in CTkTextbox is not working well.

### DISCUSSION THREAD

**Discussion by mossan0101** [TomSchimansky/CustomTkinter#1879] [Q&A] [answered]
```
import customtkinter as ctk


class App(ctk.CTk):
    def __init__(self, *args, **kwargs):
        super().__init__(*args, **kwargs)

        self.geometry("1000x600")

        self.info_textbox = ctk.CTkTextbox(self, state="disabled", width=800, height=500)
        self.info_textbox.grid(row=0, column=0, padx=6, pady=6)

        AA_list = [
            "   _____       ______", 
            "  / ___/____  / __/ /__      ______ _________",
            "  \__ \/ __ \/ /_/ __/ | /| / / __ `/ ___/ _ \\",
            " ___/ / /_/ / __/ /_ | |/ |/ / /_/ / /  /  __/",
            "/____/\____/_/  \__/ |__/|__/\__,_/_/   \___/",
        ]
                  
        for line in AA_list:
            self.writeinfo(f"{line}\n")


    def writeinfo(self, string):
        self.info_textbox.configure(state="normal")
        self.info_textbox.insert("end", string)
        self.info_textbox.see("end")
        self.info_textbox.configure(state="disabled")

app = App()
app.mainloop()
```

I am thinking of writing characters in a text box to check the status of the processing. At that time, I want to display the ASCII art of the program first, but with this code, some parts are collapsing like in the image. I think it’s probably because the size of the space character is smaller than the normal character. Is there any way to display it normally?

![test](https://github.com/TomSchimansky/CustomTkinter/assets/93064942/584ccc1e-3e02-43cf-8cc9-16f928e583b4)

**Comment by Akascape** [comment]
@mossan0101 Try experimenting with the font sizes.

**Reply by avalon60** [reply, marked as answer]
I am guessing that this because you are defaulting to a proportional font, when inside CustomTkinter.

**Reply by mossan0101** [reply]
![test2](https://github.com/TomSchimansky/CustomTkinter/assets/93064942/b52925be-25ca-414f-bd21-162e65572389)

I changed the code to 
```
self.info_textbox = ctk.CTkTextbox(self, state="disabled", font=("Consolas", 20), width=800, height=500)
```
 and it worked well! I learned for the first time that there are different types of fonts such as proportional font and monospaced font. Thank you for your advice.
